src/pages/layer-2.js

```diff
@@ -0,0 +1,629 @@
+        pathId: "/",
+        isSecondary: "isSecondary",
+      },
+    ],
+  }
+
+  const layer2Cards = [
+    {
+      image: privacyGlyph,
+      title: "Inclusivity",
+      preview: "Lower fees and better user experience",
+      body: [
+        "By combining multiple transactions into a single transaction on layer 1, transaction fees are massively reduced, making Ethereum more accessible for all.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Expand Use Cases",
+      preview: "Increase the scope of projects",
+      body: [
+        "With higher transactions per second, cheaper fees, and new technology, the scope of projects is able to expand into new use cases we have yet to see come online. Opportunities for gaming, exchanges, and applications yet to be created will be able to come online with better user experience.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Security",
+      preview: "Layer 2 is cashing a cheque, layer 1 is going to court",
+      body: [
+        "We need need scaling solutions that inherit Ethereums security. Layer 2 networks settle their transactions on the Ethereum mainnet, allowing users who use them to benefit from the security of the Ethereum network.",
+        "Layer 2s take advantage of the consensus mechanism of its parent Layer 1, in this case Ethereum, instead of providing their own consensus mechanism.",
+      ],
+      alt: "test",
+    },
+  ]
+
+  const layer2DataCombined = [...layer2Data.optimistic, ...layer2Data.zk]
+
+  return (
+    <GappedPage>
+      <PageMetadata
+        title={"Layer 2"}
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <Content>
+        <h2>What is Layer 1?</h2>
+        <p>
+          Before diving into "layer 2", it helps to understand what we consider
+          "layer 1".
+        </p>
+        <p>
+          Layer 1 blockchains, such as Ethereum and Bitcoin, are the underlying
+          foundation that layer 2 projects build on top of. Examples of layer 2
+          projects built on top of these networks include zero-knowledge rollups
+          and optimistic rollups on Ethereum and the Lighting Network on top of
+          Bitcoin.
+        </p>
+        <p>
+          Ethereum as the layer 1 includes the blockchain itself and the history
+          of transaction data, a network of miners (soon to be validators after{" "}
+          <Link to="/upgrades/merge/">The Merge</Link>) proposing blocks, a
+          network of node operators to secure and validate the network, and the
+          consensus mechanism for the network. Ethereum also functions as a data
+          availability layer for Layer 2s, and if there are any disputes on
+          previous transactions data is provided from Ethereum for these
+          disputes.
```

Comment:
Maybe we could break this list of things into bullet points to be easier to visualize:
```
Ethereum as the layer 1 includes:
- The blockchain itself and the history of transaction data.
- A network of miners (soon to be validators after The Merge) proposing blocks.
- A network of node operators to secure and validate the network.
- The consensus mechanism for the network.

Ethereum also functions as a data availability layer for Layer 2s, and if there are any disputes on previous transactions data is provided from Ethereum for these disputes.
```

Reply:
Jakub and Nuno took this idea and ran with it, will implement on next design pass :)